evaluate_tf.py

```diff
@@ -0,0 +1,45 @@
+import tensorflow as tf
+import numpy as np
+import argparse
+from classification import Classification
+import cv2 as cv
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--graph', help='.pb graph path', default='resnet_v2_101_299_frozen.pb')
+    parser.add_argument('--labels', help='.txt labels path', default='classification_classes_ILSVRC2012.txt')
+    parser.add_argument('--databasePath', help='database path', required=True)
+    parser.add_argument('--rightVals', help='.txt val path', default='val.txt')
+    argv = parser.parse_args()
+
+    graph = Classification(argv.graph)
+
+    with open(argv.labels, 'rt') as f:
+        labels = f.read().strip().split('\n')  
+
+    
+    with open(argv.rightVals, 'rt') as f:
+        vals = f.read().strip().split('\n')
+
+    top_1 = 0
+    top_5 = 0
+    size = len(vals)
+
+    for i in range(size):
+        imgPath = argv.databasePath + vals[i].rsplit(' ')[0]
+        img = cv.imread(imgPath)
+        val = int(vals[i].rsplit(' ')[1])
+        results = graph.classify(img)
+
+        if val == (results[0] - 1):
+            top_1 += 1
+
+        if val in (results - 1):
```

Comment:
What is `- 1`?

Reply:
I think, the ```val.txt``` is wrong for ```classification_classes_ILSVRC2012.txt```. Numbers of classes differ by 1